src/threading/thread.h

```diff
@@ -119,6 +119,11 @@ class Thread {
 	 */
 	bool setPriority(int prio);
 
+	/*
+	 * Returns the current thread object if it exists.
```

Comment:
```suggestion
	 * Returns the thread object of the current thread object if it exists.
```

Reply:
```suggestion
	 * Returns the thread object of the current thread if it exists.
```

I guess you meant this?

Reply:
yes